CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+1. How reliable are those steps to reproduce the issue?
+1. Describe the outcome of those steps.
+1. Then describe what would have been expected and why.
+1. Don't make vague reports. If it is performances related, include numbers.
+
+### Following-up on the report
+
+* It's OK to ping *once* or *twice*.
+* But give time to the community to reproduce and come up with a solution.
+* Also note we all have limited time and prioritize things.
+
+## Suggesting new features
+
+Features requirement vary depending on users, use cases, priorities, etc. But
+expressing an interest is valuable as it can help prioritizing things and
+shaping the actual behavior.
+
+### Before suggesting a new feature
+
+1. Check the feature isn't already available. See the tool help.
+1. Check it is not already being planned by looking at the different
+   [milestones](https://github.com/retis-org/retis/milestones) and
+   [issues](https://github.com/retis-org/retis/issues). If a matching issue is
+   found, please comment on it to express an interest and make suggestions.
+
+### Adding an issue to request a new feature
+
+1. Use a clear, concise and descriptive title.
+1. Describe the environment on which the feature would be used and provide
+   value.
+1. Provide a detailed explanation of what you'd like to be implemented, the
+   expected output and behavior.
+1. If applicable, explain what the current behavior is lacking and how different
+   you'd like it to be.
+1. Understand we have limited time and do prioritize things. In this regard,
+   external contributions for new features are highly recommended and can have a
+   great impact in choosing the direction of the project.
+
+## Code contribution
+
+Looking for topics to work on? Please have a look at issues triaged as
+[good first issues](https://github.com/retis-org/retis/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue")
+and/or [help wanted](https://github.com/retis-org/retis/issues?q=is%3Aissue+is%3Aopen+label%3A"help+wanted").
+
+### Guidelines
+
+1. Follow the [coding style](#coding-style).
+1. Document functions, structures (including members), enums, ..., and the code
+   itself. You can look at the existing code for understanding what level of
+   documentation we expect.
+1. Add unit tests.
+1. Base new work on top of the `main` branch.
+
+### Coding style
+
+We strictly follow the Rust coding style as enforced by `rustfmt` and have a
+strong preference for the
+[Linux kernel](https://www.kernel.org/doc/html/latest/process/coding-style.html)
+[networking](https://www.kernel.org/doc/html/latest/process/maintainer-netdev.html#multi-line-comments)
+coding style for the BPF parts.
+
+### Opening a pull-request
+
+1. Before opening the pull-request, test the changes. Yes, even for those simple
+   one-liner last minute changes.
+1. Check the following commands do not return an error:
+   1. `cargo fmt --check`
+   1. `cargo clippy -- -D warnings`
+   1. `cargo test`, or to include runtime tests,
```

Comment:
I'm not entirely sure, but 

`cirrus run --container-backend=podman`

can be something to consider to add as an alternative.
`--features=test_cap_bpf` still need to be tested locally

Reply:
that's why I wasn't really sure about it.
Let's just keep it as it is.